phpstan-dbal-3.neon.dist

```diff
@@ -23,3 +23,14 @@ parameters:
                 - tests/Doctrine/Migrations/Tests/Tools/Console/legacy-config-dbal/cli-config.php
                 - tests/Doctrine/Migrations/Tests/Tools/Console/legacy-config-wrong/cli-config.php
 
+        # We need to implement platform-aware comparison
```

Comment:
This seems to impactful to do in this pull request. Not sure it would qualify as a bugfix either (but then neither does addressing a deprecation if you ask me).

Reply:
Keeping track of this at https://github.com/doctrine/migrations/issues/1227